core/src/main/java/org/pentaho/platform/scheduler2/action/ActionRunner.java

```diff
@@ -70,6 +70,9 @@ public class ActionRunner implements IActionRunner {
   public static final String KEY_USE_JCR = "useJcr"; // TODO move to more common place
   public static final String KEY_JCR_OUTPUT_PATH = "jcrOutputPath"; // TODO move to more common place
 
+  public static final String EXTERNALIZE_STYLE = "externalize-style";
```

Comment:
Only thing I wouldn't add these variables #EXTERNALIZE_STYLE and #BASE64_IMAGES since they aren't being used and it looks like the default behavior is disabled based on https://github.com/pentaho/pentaho-platform-plugin-reporting/pull/852/files